interfaces/acados_matlab/acados_ocp.m

```diff
@@ -12,42 +12,42 @@
 	methods
 
 
-		function obj = acados_ocp(model, opts)
+		function obj = acados_ocp(model, opts, build_dir)
```

Comment:
I would rather add `build_dir` in the `acados_ocp_opts` (where e.g. there are already the flags to enable/disable code generation and mex compilation)

Reply:
moved to acados_ocp_opts

Reply:
i named the option `output_dir`

Reply:
great!